rust/src/lib.rs

```diff
@@ -739,15 +783,22 @@ pub unsafe extern "C" fn zcashlc_get_next_available_address(
     db_data_len: usize,
     account_uuid_bytes: *const u8,
     network_id: u32,
+    receiver_flags: u8,
```

Comment:
I'm a little uncomfortable adding a `u8` bitflag with half of its values already assigned. I can imagine a minimum of two more kinds of receivers we would want in future (long-term storage, and PQ-safe), potentially more if we add receiver types with e.g. detection keys. I would suggest making this at least a `u16` from the get-go.

Reply:
In that case I'll just go to u32, since it will come across the FFI as i32 anyway.

Reply:
Oh wait, I'm thinking of Android. But in any case, I'll use u32 for parity with Android.